<!-- 
This launch file starts soundplay_node.py, which is the node that plays
sounds based on messages on the robotsound topic.  
-->

<launch>
  <arg name="device" default="hw:1,0" />
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py">
    <param name="device" value="$(arg device)" />
  </node>
</launch>
